src/components/pages/ImageClassification/ImageAnnotationModal/useZoomToPointsByAttributeId.js

```diff
@@ -1,28 +1,21 @@
 // importing from @turf/turf causes issues with tests, so import each utility individually
 import bbox from '@turf/bbox'
-import buffer from '@turf/buffer'
 
-export const useZoomToPointsByAttributeId = ({ getPointsGeojson, mapRef }) => {
+export const useZoomToPointsByAttributeId = ({ patchesGeoJson, zoomToPaddedBounds }) => {
   const zoomToPointsByAttributeId = (attributeId) => {
-    // we need to zoom out first to reset the bounds so that zooming in works properly
-    // since the points geojson is created using the maps current bounds
-    mapRef.current.setZoom(0)
+    if (!patchesGeoJson.features.length) {
```

Comment:
Do you need to handle `patchesGeoJson` being nullish here?

Reply:
Joe = `tsc` :)